serialmfg/serial_resources/component_instance.py

```diff
@@ -105,14 +106,14 @@ def get(identifier):
         return ComponentInstance(returned_instances[0])
     
     @staticmethod
-    def create(identifier, component_name, part_number_id=None):
```

Comment:
This `part_number_id` was for internal use only until now right? Our customers previously did not have access to this UUID through the UI or API so we can be confident that we're not pushing a breaking change? Is that a correct assumption?